main.go

```diff
@@ -75,20 +76,24 @@ func ToUrl(identifier string) (string, string, string, error) {
 
 func main() {
   c := &Config{}
+	logger := log.New()
+	logger.SetFormatter(&log.JSONFormatter{})
 
   // Load configruation from environment variables
   err := godotenv.Load(".env")
   if err != nil {
-    fmt.Println("Failed to load .env file")
+		logger.Fatalf("Failed to load .env file: %v", err)
```

Comment:
In production there is no .env file, this should be info level, certainly not Fatal

Reply:
Done!